picard/tagger.py

```diff
@@ -412,7 +412,32 @@ def process(result=None, error=None):
                     self.add_files(result)
                 thread.run_task(get_files, process)
 
-        process(True, False)
+        def get_files_non_recursive():
+            listings = [f for f in os.listdir(path)]
+            if ignore_hidden:
+                listings = filter(lambda x: not is_hidden(x), listings)
```

Comment:
This is not correct. `is_hidden` requires the full path to work correctly on all platforms, but x is only the filename at this point.